### ISSUE TITLE
Not idempotent (extra newline every time)

### ISSUE THREAD

**Issue by jkozdon** [kdheepak/JuliaFormatter.vim#9]
Each time `call JuliaFormatter#Format(0)` is called an extra newline is added to the end of the file.

## Troubleshooting

vim info:
```
VIM - Vi IMproved 8.1 (2018 May 18, compiled Dec 29 2019 00:36:15)
```

julia info:
```
julia> versioninfo()
Julia Version 1.3.1
Commit 2d5741174c (2019-12-30 21:36 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin18.6.0)
  CPU: Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, haswell)
Environment:
  JULIA_EDITOR = vim
  JULIA_PKG_DEVDIR = /Users/jekozdon/codes/julia_dev
```

**Comment by kdheepak** [comment]
You are right! I hadn't even noticed that. I've fixed it in 8dd2d4345f258e3db8d294e84467ba41d7fd4349.

**Comment by jkozdon** [comment]
Thanks.
